library.gradle

```
@@ -18,8 +18,7 @@ android {
         minSdkVersion minVersion
         targetSdkVersion compileVersion
         versionCode 1
-        versionName "5.6.0"
-
+        versionName "5.7.0-SNAPSHOT"
```

I would leave the empty line as it was.
```suggestion
        versionName "5.7.0-SNAPSHOT"

```

---

🕺